.github/workflows/VeniceCI-CompatibilityTests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 60
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Can we extract out the version into a variable to keep the changes minimal for the future edits? e.g., would something like this https://docs.geoserver.org/2.21.x/en/user/styling/ysld/reference/variables.html help extract out the actions or the version into a variable?

Reply:
Some of these files are generated by our gradle build, so we could inject the variable there also. In any case, I don't think this changes very often. So IDK if it's worth it 🤷‍♂️

Reply:
ah.. You can ignore the comment for now. I do see there are plenty of workflows and steps that are copied across the yaml we have. Maybe this can be a nice to do clean up to consolidate how these workflows are in general put together and modified.

Perhaps, finding a way to clean up and organize these systematically would be nicer.